assets/backport/main.go

```
@@ -0,0 +1,149 @@
+
+		// Create the pull request.
+		if err = clt.CreatePullRequest(ctx, targetBranch, newBranchName, prNumber); err != nil {
+			log.Fatal(err)
+		}
+		fmt.Printf("Pull request created for branch %s.\n", newBranchName)
+	}
+	fmt.Println("Backporting complete.")
+}
+
+type Config struct {
+	Host Host `yaml:"github.com"`
+}
+
+type Host struct {
+	Token string `yaml:"oauth_token"`
+}
+
+// githubConfigPath is the default config path
+// for the Github CLI tool.
+const githubConfigPath = ".config/gh/hosts.yml"
+
+// getGithubToken gets the Github auth token from
+// the Github CLI config.
+func getGithubToken() (string, error) {
+	dirname, err := os.UserHomeDir()
+	if err != nil {
+		return "", trace.Wrap(err)
+	}
+	ghConfigPath := filepath.Join(dirname, githubConfigPath)
+	yamlFile, err := ioutil.ReadFile(ghConfigPath)
+	if err != nil {
+		return "", trace.Wrap(err)
+	}
+
+	var config *Config = new(Config)
+
+	if err = yaml.Unmarshal(yamlFile, config); err != nil {
+		return "", trace.Wrap(err)
+	}
+	if config.Host.Token == "" {
+		return "", trace.BadParameter("missing Github token.")
+	}
+	return config.Host.Token, nil
+}
+
+// parseFlags parses flags and sets
+func parseFlags() ([]string, int, string, string, error) {
+	var (
+		to    = flag.String("to", "", "List of comma-separated branch names to backport to.\n Ex: branch/v6,branch/v7\n")
+		pr    = flag.Int("pr", 0, "Pull request with changes to backport.")
+		owner = flag.String("owner", "gravitational", "Name of the repository's owner.")
+		repo  = flag.String("repo", "teleport", "Name of the repository to open up pull requests in.")
+	)
+	flag.Parse()
+	if *to == "" {
+		return nil, 0, "", "", trace.BadParameter("must supply branches to backport to.")
+	}
+	if *pr == 0 {
+		return nil, 0, "", "", trace.BadParameter("much supply pull request with changes to backport.")
+	}
+	// Parse branches to backport to.
+	backportBranches, err := parseBranches(*to)
+	if err != nil {
+		log.Fatal(err)
```

Return an error here instead of `log.Fatal`.